import util



class error(Exception):
    pass

class DebuggerException(error):
    pass
